readme.md

```diff
@@ -19,6 +19,18 @@ stripAnsi('\u001B[4mUnicorn\u001B[0m');
 //=> 'Unicorn'
 ```
 
+If you want to support streams you can use [`replacestream`](https://github.com/eugeneware/replacestream) like below:
+
+```js
+const fs = require('fs');
+const replacestream = require('replacestream');
+const stripAnsi = require('strip-ansi');
```

Comment:
You meant,

```js
const ansiRegex = require('ansi-regex');
```

Reply:
Oh, yeah. Should it really be documented here then?